crates/python/src/api.rs

```diff
@@ -102,6 +102,16 @@ py_wrap_error!(
     PyRuntimeError
 );
 
+/// Get the keyword `key` value from `kwds` if it is of type `Option<T>`, else `None`.
+/// Note that values present at `key` that cannot be extracted into `Option<T>` are discarded.
```

Comment:
This seems more accurate?
```suggestion
/// Note that values present at `key` that cannot be extracted into `Option<T>` are treated as `None`.
```